index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <meta charset='utf-8'>
+   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+   <title>My Portfolio</title>
+   <meta name='viewport' content='width=device-width, initial-scale=1'>
+   <link rel='stylesheet' type='text/css' media='screen' href='styles.css'>
+   <script src='main.js'></script>
+</head>
+<body>
+   <header>
+      <div class="logo">
+         <h3>My Logo</h3>
```

Comment:
- [ ] Kindly use `a` tag instead of `h3` tag. It is the convention to use `a` tag for logos or title in the header/navigation section of a web page. Using `a` tag is also good for semantic purposes and it provides a great user experience. Kindly read this [article](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more details. 👍